docs/standard/base-types/type-conversion.md

```diff
@@ -5,7 +5,7 @@ ms.date: "03/30/2017"
 ms.prod: ".net"
 ms.reviewer: ""
 ms.suite: ""
-ms.technology:v
+ms.technology: v
```

Comment:
@BradleyBartlett Do u want to set it to `dotnet-standard`? If not, no worries. We'll square it away on our end.